src/redux/actions/permission-action.js

```diff
@@ -16,3 +16,25 @@ export const listPermissionOptions = ({ field, limit, offset, application, resou
   type: fieldToAction[field],
   payload: PermissionsHelper.listPermissionOptions(field, limit, offset, application, resourceType, verb, options),
 });
+
+export const expandSplats = ({
+  limit = 1000,
```

Comment:
does this endpoint ignore the max API limit of 100?

Reply:
When I tried it in API docs, it seems that 1000 is the limit.

Reply:
Nice good to know :+1: